renderer_elements.js

```diff
@@ -82,7 +115,7 @@ class SDFG extends SDFGElement {
 }
 
 class State extends SDFGElement {
-    draw(renderer, ctx, mousepos) {
+    draw(renderer, ctx, mousepos, debug_draw = false) {
```

Comment:
there must be an easier way without overriding every class' draw and draw_simple methods...

Reply:
Technically `debug_draw` can be read as a global inside `SDFGElement`'s `debug_draw` method, but I believe that still requires a super call. Alternatively, all the `debug_draw` calls can be explicitly made from within `draw_sdfg`?

Reply:
The latter may work better, yes.